docs/docs/modeling.md

```diff
@@ -116,6 +116,20 @@ classDiagram
 
 ## TenantsWork -
 
-## AssignedCategory -　
+## AssignedCategory -　割当カテゴリー
+
+Category assigned to each tenant.
```

Comment:
It might be better to mention that this `AssignedCategory` is referred from/based on `Category`, or the `AssignedCategory` cannot be created without any `Category`.